src/images.ts

```diff
@@ -6,7 +6,7 @@ import { AttachmentsResolver } from '@jupyterlab/attachments';
 
 type Options = {
   resolver: IRenderMime.IResolver | null;
-  cell: MarkdownCell;
+  cell?: MarkdownCell;
 };
 
 export async function imageUrlSourceTransform(
```

Comment:
This function was updated to remove the cell as a requirement, which has an attachments model.